stan/math/fwd/fun/log_mix.hpp

```diff
@@ -28,7 +28,6 @@ template <typename T_theta, typename T_lambda1, typename T_lambda2, int N>
 inline void log_mix_partial_helper(
     const T_theta& theta, const T_lambda1& lambda1, const T_lambda2& lambda2,
     promote_args_t<T_theta, T_lambda1, T_lambda2> (&partials_array)[N]) {
-  using std::exp;
```

Comment:
This function actually calls exp, so I think we should include it from fwd?

Reply:
👍

Reply:
👍